pr_3/javascript/random_samples/insufficiently_protected_credentials.js

```
@@ -0,0 +1,27 @@
+function insufficientlyProtectedCredentialsNoncompliant() {
+    User.findOne({ email: req.body.email }, function (e, user) {
+        // Noncompliant: object is passed directly to `jsonwebtoken.sign()`.
+        var token = jwt.sign(user, key, { expiresIn: 60 * 60 * 10 })
+        return token
+    })
+}
+// {/fact}
+
+
+// {fact rule=insufficiently-protected-credentials@v1.0 defects=0}
+var jwt = require("jsonwebtoken")
```

Biome (2.1.2)

[error] Shouldn't redeclare 'jwt'. Consider to delete it or rename it.

'jwt' is defined here:


(lint/suspicious/noRedeclare)